c_common/models/system_models/src/extra_monitor_support.c

```diff
@@ -699,9 +672,9 @@ static inline int reinjection_get_status(sdp_msg_t *msg) {
             (reinjector_status_response_packet_t *) &msg->arg1;
 
     // Put the router timeouts in the packet
-    uint control = (uint) (rtr[RTR_CONTROL] & 0xFFFF0000);
-    data->router_timeout = (control >> 16) & ROUTER_TIMEOUT_MASK;
-    data->router_emergency_timeout = (control >> 24) & ROUTER_TIMEOUT_MASK;
+    router_control_t control = router_control->control;
```

Comment:
this going to also sound stupid. but are they not the wrong way around?
timeout = emergency wait             vs             timeout = wait
emergency timeout =wait.                              emergency timeout =  emerngecy wait

Reply:
At the hardware level, they're really `wait0` and `wait1` I think. The `wait0` is the time the hardware will wait between the packet arriving in the router's buffer and it being switched to emergency handling (which SCAMP actually never configures, FWIW). The `wait1` is the time between when the router switches to emergency handling and when the packet is dumped.

Probably. This could be a whole lot clearer. _The datasheet is not especially clear about the naming of things in this area either._

Reply:
Yes that sounds right.

Reply:
Correction, they're `wait1` and `wait2`. It doesn't really matter all that much what the labels are.

Reply:
I ought to change the documentation of the Python code to talk to the extra monitor. That's where things really are misleading.

Reply:
See #624 for my thoughts on that. That PR is _strictly_ downstream of this one.